src/components/KeyControlledList/KeyControlledList.js

```diff
@@ -106,7 +107,7 @@ export default class KeyControlledList extends React.Component {
   render () {
     const { selectedIndex } = this.state
 
-    const { theme, children, ...props } = this.props
+    const { tagType, theme, children, ...props } = this.props
```

Comment:
it would be nice if we could make this work in a more generic way since this is meant to be a generic component, so instead of tagType have props for headerText, and for countText, then check if the item has a .count field to know whether to show a count or not. but if that all is a pain then lets just document that we would like to change that in the future. like the FIXME comment that already exists here

Reply:
If it's ok with you, I'll just write a FIXME note for now, and I'll address it when I get to improvements on Topics/Tags

Reply:
sure